src/cutadapt/__main__.py

```diff
@@ -286,6 +285,10 @@ def get_argument_parser():
             "depending on input. The summary report is sent to standard output. "
             "Use '{name}' in FILE to demultiplex reads into multiple "
             "files. Default: write to standard output")
+    group.add_argument('--compression-level', default=6, dest="compresslevel",
+        help= 'Compression level if gzipped output files are used.  Default: %(default)s')
+    group.add_argument('-Z', action='store_true', dest='Z',
+        help= 'Short-hand for `--compression_level 1`. ')
```

Comment:
As this isn’t going to be rendered as Markdown, please don’t use backticks here (just use no quotation marks at all). Please also write `--compression-level=1` instead, please (hyphen instead of underscore and an equals sign). (And there’s also a trailing space that needs to be removed.)